core/src/net/mod.rs

```diff
@@ -645,8 +649,17 @@ pub mod net_test_helpers {
                 target,
                 count: 0,
                 eager: true,
+                promise: None,
             }
         }
+
+        /// Creates a future which will be completed by the Pinger when it has received pongs for
+        /// all the pings it sent
+        pub fn make_future(&mut self) -> KFuture<()> {
```

Comment:
Should we perhaps name this function `completion_future` or something to that effect? The return type already kinda implies that it "makes a future" but giving it a good name might obviate the need for the doc.